draft-ietf-quic-transport.md

```diff
@@ -541,12 +721,54 @@ packet number.  Once any packet has been acknowledged, subsequent packets can
 use a shorter packet number encoding.
 
 
-## Frames and Frame Types {#frames}
+## Handling Packets from Different Versions {#version-specific}
+
+Between different versions the following things are guaranteed to remain
+constant:
+
+* the location and size of the Flags field in both header forms,
```

Comment:
You didn't originally call that field Flags in this version, though I see it has cropped up other places through the document.